doc/nrf/links.txt

```diff
@@ -616,6 +616,7 @@
 .. _`nRF Connect for Cloud Patch Device State`: https://api.nrfcloud.com/v1#operation/UpdateDeviceState
 .. _`nRF Cloud REST API (v1)`:
 .. _`nRF Cloud Device API`: https://api.nrfcloud.com/v1
+.. _`nrf Cloud Device Messages`: https://docs.nrfcloud.com/Reference/DeviceMessages/
```

Comment:
```suggestion
.. _`nRF Cloud Device Messages`: https://docs.nrfcloud.com/Reference/DeviceMessages/
```

Reply:
🤦I swear I will stop making this mistake eventually

Thanks for catching that!